### DISCUSSION TITLE
Should volume work on a MacBook Pro 2019 (Monterey 12.6) and an Apple Studio Display 27"

### DISCUSSION THREAD

**Discussion by rob-derosa** [MonitorControl/MonitorControl#1232] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

Volume controls don't show despite being enabled in Preferences. Thanks!

**Comment by waydabber** [comment, marked as answer]
The Apple Studio Display 27" should have native volume control at OS level. MonitorControl won't treat the display as DDC compatible so it won't offer volume controls for it!
